src/Maker/MakeForm.php

```diff
@@ -41,6 +51,25 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
             ->addArgument('name', InputArgument::OPTIONAL, sprintf('The name of the form class (e.g. <fg=yellow>%sType</>)', Str::asClassName(Str::getRandomTerm())))
             ->setHelp(file_get_contents(__DIR__.'/../Resources/help/MakeForm.txt'))
         ;
+
+        $inputConf->setArgumentAsNonInteractive('name');
```

Comment:
yes it can be interactive and it was interactive. There were some problems with detecting if user entered `name` as argument. I need it to fast form generation with only one argument. 

I think if we add a second argument it will be more simple.

Reply:
I think my preference would be: 2 arguments (`name` and `bound-class`). Then I think we can let the base maker command do all the normal interactive stuff for us. Actually, we could make `bound-class` an option - e.g .`--bound-class`, as it might make passing "nothing" easier if you don't want one: e.g. `--bound-class=`.

Btw, in general, I'm not very concerned with making the commands work in a *non* interactive state. It causes too much complexity.

Reply:
Well actually, you might still need to ask the `bound-class` value manually in `interact` so we can do some auto-completion :)